docs/lib/storage/fragments/js/upload.md

```diff
@@ -95,17 +95,93 @@ async function onChange(e) {
 Upload an image in your React Native app:
 
 ```javascript
-async function pathToImageFile() {
-  try {
-    const response = await fetch(pathToImageFile);
-    const blob = await response.blob();
-    await Storage.put('yourKeyHere', blob, {
-      contentType: 'image/jpeg', // contentType is optional
+async function pathToImageFile(data) {
+      try {
+        const response = await fetch(data);
+        const blob = await response.blob();
+        await Storage.put(`customers/${name}`, blob, {
+          contentType: 'image/jpeg', // contentType is optional
+        });
+      } catch (err) {
+        console.log('Error uploading file:', err);
+      }
+    }
+    pathToImageFile(result.uri);
+  }
+```
+
+
+## React Native upload images with Expo Image Picker
```

Comment:
Since a lot of people are using RN and Expo, I think it's important to link this code here.

Reply:
Have a couple concerns:
- we are not supplying a vanilla RN example that parallels the expo example, can `expo-image-picker` be used without expo?
- any API changes to `expo-image-picker` will not be reflected in our docs